docs/course_materials/functions.html

```diff
@@ -113,6 +113,21 @@ <h3><b>Variables Scope</b></h3>
 print(a)   # 5
                     </code></pre>
                 </section>
+
+              <section>
+                <h3><b>return statement</b></h3>
```

Comment:
Return

Reply:
but the return statement should be lower case

Reply:
maybe "The return statement"?